presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -80,8 +80,17 @@
     private static final ObjectMapper JSON_PARSER = new ObjectMapperProvider().get();
 
     private static final String PASSTHROUGH_QUERY_SUFFIX = "$query";
-    private final Map<String, ColumnHandle> queryTableColumns;
-    private final ColumnMetadata queryResultColumnMetadata;
+
+    private static final Map<String, ColumnHandle> PASSTHROUGH_QUERY_COLUMNS = ImmutableMap.of(
+            "result",
```

Comment:
`"result"` is appearing multiple times. Please either extract contant. 
Or move definition of `PASSTHROUGH_QUERY_RESULT_COLUMN_METADATA` above `PASSTHROUGH_QUERY_COLUMNS`.
And the use `PASSTHROUGH_QUERY_RESULT_COLUMN_METADATA.getName()` in definition of `PASSTHROUGH_QUERY_COLUMNS`.